cvat/apps/iam/permissions.py

```diff
@@ -55,6 +55,8 @@ def get_organization(request, obj):
     # Try to get organization from an object otherwise, return the organization that is specified in query parameters
     if obj is not None and isinstance(obj, Organization):
         return obj
+    if obj is not None and isinstance(obj, AnnotationGuide):
+        return (obj.project if obj.project else obj.task).organization
```

Comment:
```suggestion
```
let's use the `organization_id` property for AnnotationGuide model that I suggested above

Reply:
Done